docs/django/backend.rst

```diff
@@ -0,0 +1,56 @@
+Django Email Backend
+====================
+
+SparkPost python library comes with an email backend for Django.
+
+Configure Django
+----------------
+
+To configure Django to use SparkPost email, put the following configuration in `settings.py` file.
+
+.. code-block:: python
+
+    SPARKPOST_API_KEY = 'API_KEY'
+    EMAIL_BACKEND = 'sparkpost.django.email_backend.SparkPostEmailBackend'
+
+Replace *API_KEY* with an actual API key.
+
+
+Sending an email
+----------------
+
+As Django is now configured to use SparkPost's email backend, we can send email simply, as you usually do, using its `send_mail` method.
```

Comment:
Copy edit:

`Django is now configured to use the SparkPost email backend. You can now send mail using Django's send_mail method:`